src/assets/scripts/client/aircraft/StripView/StripViewController.js

```diff
@@ -132,14 +132,11 @@ export default class StripViewController {
             // is shown immediately. For arrivals, this does not work so well. We need to `$.detach() the
             // strip and re-add it to the list so it is at the end of the list.
             if (aircraftModel.inside_ctr && !stripViewModel.insideCenter) {
-                stripViewModel.$element.detach();
```

Comment:
So is the above TODO resolved by removing the `.detach()`?

```js
// TODO: this should be looked at again
// an aircraft strip is created on instantiation, which works for departures where a strip
// is shown immediately. For arrivals, this does not work so well. We need to `$.detach() the
// strip and re-add it to the list so it is at the end of the list.
```

Reply:
done